test/fast/Running "nvm unload" should unset all function and variables.

```diff
@@ -5,7 +5,7 @@ set -ex
 BEFORE="./before.tmp"
 AFTER="./after.tmp"
 
-cleanup () { echo rm -f "${BEFORE}" "${AFTER}"; }
+cleanup () { rm -f "${BEFORE}" "${AFTER}"; }
```

Comment:
ha, no this was just be debugging locally and forgetting to remove my `echo`. good catch.